### PR TITLE
Add missing job detail fields

### PR DESCRIPTION
**Pull request by marshmalien**
##### SUMMARY
This PR adds the following missing job detail items:
- Job Status icon
- Launched By label and link 
- Project status icon
- Playbook value
- Environment value
- Execution Node value

Other items addressed: 
- Move the large mock job json file to a shared directory under Job since the JobDetail and JobOutput test files both use this mock job.  
- Update OrganizationAccessItem snapshot since the DetailList alignment style changed
- Update JobDetail unit tests for the newly added job detail fields

##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
 - UI

##### ADDITIONAL INFORMATION

_Before_ 
Screen Shot 2019-09-24 at 3 28 26 PM

_After_
Screen Shot 2019-09-24 at 3 29 02 PM

### CODE REVIEW

File: awx/ui_next/src/screens/Job/JobDetail/JobDetail.jsx [hunk 1]
```diff
@@ -21,6 +22,14 @@ const VariablesInput = styled(_VariablesInput)`
   }
 `;
 
+const StatusDetailValue = styled.div`
+  align-items: center;
+  display: inline-flex;
+  > div {
```
Comment (keithjgrant) [inline awx/ui_next/src/screens/Job/JobDetail/JobDetail.jsx]: The generic `div` selector here is likely to target unwanted other stuff that might get added later on. Can you either put a class on the div in question or use styled components to apply this margin directly to it? (I think this `div` targets the `StatusIcon`?)

**Review by keithjgrant — APPROVED** [review]
Looks great! Added one suggestion in the comments, but otherwise 👍🏻

File: awx/ui_next/src/screens/Job/JobDetail/JobDetail.jsx (hunk 1 above)
Comment (marshmalien) [reply awx/ui_next/src/screens/Job/JobDetail/JobDetail.jsx]: Good point - I'll push up a commit to add a class to the StatusIcon wrapper 👍

**Comment by marshmalien** [comment]
regate

**Review by AlexSCorey — CHANGES_REQUESTED** [review]
If somehow a job is created but doesn't have a `launchedByValue` the `Launched By` label still displays on the page but there is no value.  I think if you remove `<>` on line 112 and `</>` on line 118 that should take care of the problem.

**Review by AlexSCorey — APPROVED** [review]

**Pull request merged**